types/react-select/index.d.ts

```diff
@@ -228,7 +228,12 @@ export interface ReactSelectProps<TValue = OptionValues> extends React.Props<Rea
     /**
      * method to filter the options array
      */
-    filterOptions?: FilterOptionsHandler<TValue>;
+    filterOptions?: FilterOptionsHandler<TValue>;                                      
+    /**
+     * id for the underlying HTML input element
+     * @default ""
```

Comment:
Why does `id` prop have empty string as default value instead of `undefined` like in [react-select docs](https://github.com/JedWatson/react-select/blob/82910d9225f7aff73ac5052da9dc1e7f52f13de7/README.md#select-props)?

Reply:
Ah, that's my bad. Updated it.